src/guacamole-docker.md

```diff
@@ -646,6 +646,41 @@ valid Docker variables for enabling and configuring header authentication:
   header that will be used used to authenticate the user to Guacamole. If this
   is not specified the default value of REMOTE_USER will be used.
 
+(guacamole-docker-tomcat-remote-ip-valve)=
+
+### Execution behind a proxy
+
+To run Guacamole behind a reverse proxy, the Tomcat's [`RemoteIpValve`](https://tomcat.apache.org/tomcat-8.5-doc/config/valve.html#Remote_IP_Valve) as in [](tomcat-remote-ip).
+
+(guacamole-docker-tomcat-remote-ip-valve-required-vars)=
+
+#### Required environment variables
+
+The following environment variables have to be set in order to configure the [`RemoteIpValve`](https://tomcat.apache.org/tomcat-8.5-doc/config/valve.html#Remote_IP_Valve):
+
+`PROXY_ALLOWED_IPS_REGEX`
+: The regex indicating the hosts allowed to set the remote IP via headers.
+  Specify `.*` to allow any address.
+  This maps with Tomcat's `internalProxies` directive.
```

Comment:
I feel like this section needs to be tuned one way or the other. In particular:
* Just telling people to "specify `.*` to allow any address", without any warning or additional information about why they'd want to do that (or why they may not want to do that) seems to me like there's an implied recommendation to go that route. If we're going to provide examples, that's fine, but let's provide more than just, "Here's how to open it up to allow anything."
* Saying that "This maps with Tomcat's `internalProxies` directive" has very little meaning, here. I would recommend that we either just point people to the Proxy section of the manual that contains the detail and leave out details, here, or that we build out the detail here a little more to match the Proxy section so that this makes complete sense for someone configuring Guacamole in Docker.